src/sentry/similarity/__init__.py

```diff
@@ -78,7 +78,7 @@ def _make_index_backend(cluster=None):
             return index
 
     return MetricsWrapper(
-        RedisMinHashIndexBackend(
+        RedisScriptMinHashIndexBackend(
```

Comment:
Probably didn't need to make this change yet

Reply:
My plan is to add another backend that doesn't depend on Lua scripting and use the base tests to ensure parity between the existing implementation and the new one